src/data/community.json

```diff
@@ -97,5 +99,19 @@
                 "external": true
             }
         }
+    },
+    "section-social-media": {
+        "title": "Social Media",
+        "anchor": "social-media",
+        "textblock": ["You can also find the most important information about the Corona-Warn-App, its features and its development on social media."],
+        "socialmedia": [{
+            "title": "Twitter",
+            "textblock": ["The Corona-Warn-App has its own <a href='https://twitter.com/coronawarnapp' target='_blank'>Twitter-Account</a>. On Twitter, we regularly inform you about key figures, updates and new features. You can also participate in polls and discuss with other followers about the Corona-Warn-App.",
+            "Follow the account, become a part of the Twitter community and help us to make the app even more known and better."]
```

Comment:
Suggest instead of:

"...and help us to make the app even more known and better."

use

"...and help us to improve the app and make it better known."

"more known" doesn't sound quite right (as a translation of "bekannter")

Reply:
Thanks @MikeMcC399! Fixed it in the latest commit.